src/sass/app/websocket_pages.scss

```diff
@@ -325,6 +339,13 @@
                 .button span {
                     width: calc(50% - 55px);
                 }
+                #btn_cancel, #btn_ok {
+                    width: calc(100% - 25px);
+
+                    span {
```

Comment:
can we use a class name selector instead of a tag name selector for efficiency?

https://css-tricks.com/efficiently-rendering-css/

Reply:
Done. Thanks for the suggestion @hojjat-fs